.gitignore

```
@@ -0,0 +1,5 @@
+__pycache__/
```

I don't think we need to commit these "__pycache__" files anywhere in the project
```suggestion
**/__pycache__/
```